src/source/geojson_wrapper.js

```diff
@@ -48,28 +42,30 @@ class FeatureWrapper implements VectorTileFeature {
         if ('id' in feature && !isNaN(feature.id)) {
             this.id = parseInt(feature.id, 10);
         }
-        this.extent = EXTENT;
     }
 
     loadGeometry() {
-        const rings = this.rawGeometry;
-        this.geometry = [];
-
-        for (let i = 0; i < rings.length; i++) {
-            const ring = rings[i],
-                newRing = [];
-            for (let j = 0; j < ring.length; j++) {
-                newRing.push(new Point(ring[j][0], ring[j][1]));
+        if (this._feature.type === 1) {
+            const geometry = [];
+            for (const point of this._feature.geometry) {
+                geometry.push([new Point(point[0], point[1])]);
             }
-            this.geometry.push(newRing);
+            return geometry;
+        } else {
+            const geometry = [];
+            for (const ring of this._feature.geometry) {
+                const newRing = [];
+                for (const point of ring) {
+                    newRing.push(new Point(point[0], point[1]));
+                }
+                geometry.push(newRing);
+            }
+            return geometry;
         }
-        return this.geometry;
     }
 
     bbox() {
-        if (!this.geometry) this.loadGeometry();
-
-        const rings = this.geometry;
+        const rings = this.loadGeometry();
```

Comment:
This will mean rebuilding the geometry from scratch on every `bbox()` call — previously it was cached. Maybe we should calculate bbox as a part of `loadGeometry` and then just return the calculated value here?

Reply:
`bbox` is never used; I removed it from the interface type to enforce that.